pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+
+In this case, since you don't know what the address of the reply channel is yet, you can either assign null to the `address` property or omit the property entirely indicating that the address is not known yet. You can now use `Operation Reply Address` object to dynamically specify where the reply has to be sent. The `Operation Reply Address` object has a property called `location` which essentially takes in a runtime expression that specifies the address of the reply channel.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example for dynamic reply channel sub-pattern
+  version: 1.0.0
+  description: Example with a requester that initiates the request/reply pattern where the reply will happen on whatever is defined in the header `replyTo` of the request.
+
+channels:
+  ping:
+    address: /ping
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: null
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      address:
+        description: The reply address is dynamically determined based on the request header `replyTo`
+        location: "$message.header#/replyTo"
+      channel: 
+        $ref: '#/channels/pong'
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      address:
+        description: The reply address is dynamically determined based on the request header `replyTo`
+        location: "$message.header#/replyTo"
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+### Request/reply over the same channel
+The request/reply pattern can also be implemented over the same channel. You can do this by specifying the same address for both the requester and the replier.
+
+Here's a example of setting up both requestor and replier in the same channel:
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with requester
+  version: 1.0.0
+  description: Requester example initiating the request-reply pattern.
+
+channels:
+  ping:
+    address: /
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: /
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+### Multiple messages over the same channel
+When working with protocols like websockets, you can have multiple messages under the same channel. In such scenarios, you can use the `messages` property in the `Operation` object to explicitly define which message among the multiple messages available over the same channel is a request and which is a reply.  
```

Comment:
let's not mention websocket, as it is not only websocket. It is more about different approach for grouping. In previous example you have 2 channels with 1 message each and the same address, and in this section you say that you can also represent the same info by having 2 channels with 2 messages and the same address. Then just mention that in an Operation you need to specify with `message` which is the requestor message and under Reply you need to also specify with `message` what is the reply message. 

During discussions when we were adding it to the spec, we called it message filtering. So that you can have channel with multiple messages on the same address, and then on operation and reply level you can use `message` to filter out the fluff and just point to specific message

Reply:
I think you missed the comment